### DISCUSSION TITLE
alf.io manual install

### DISCUSSION THREAD

**Discussion by emircanxe** [alfio-event/alf.io#1180] [Deployment]
Hello colleagues,

I don't know if my question belongs here.

Is there an Alf.io manual for the installation on Ubuntu?
I found it here:
https://gist.github.com/flexwie/a8572e8e7ef06ae844925ec857fa3b5b

However, it is very outdated and I am not getting anywhere.

I would be very grateful for any help.

**Comment by aadhilpm** [comment]
I recently deployed Alf.io for production for an event and spent a considerable amount of time understanding the concept and process. 

To help others who may be facing similar challenges, I have created a step-by-step guide on deploying Alf.io in production. 

It can be found at this link: https://gist.github.com/aadhilpm/132b272fcdd4a57b375d80178cce9a98 

I hope it will be a valuable resource for those looking to do the same.

**Reply by emircanxe** [reply]
Hey,

first of all many many thanks for this detailed tutorial!

Nevertheless, I can't get alf.io to work. I wish I knew what I'm doing wrong.

Do you think you can help me? I would be very grateful.

Your tutorial is already very detailed, but probably missing something somewhere that I do not understand as a beginner.
A video would not be feasible for you, right?

Many thanks
Emircan

**Reply by tobsowo** [reply]
Thank you for sharing the step by step.

I tried it but couldn't get it working, maybe you may want to help check out my comments.

**Comment by sysadminpower2019** [comment]
Hello, I am also having some difficulties. could we update these guides?

**Comment by sysadminpower2019** [comment]
@aadhilpm is your deployment still working?
